frontend/src/components/datasets/DatasetDetailContent.js

```diff
@@ -0,0 +1,136 @@
+import FilteredList from "../FilteredList";
+import { DATASET_FILE_FILTERS } from "../filters/descriptions";
+import PageContent from "../PageContent";
+import moment from "moment";
+
+const getTableColumns = (setReleaseFlag) => {
+    return [
+        {
+            title: "ID",
+            dataIndex: "id",
+            sorter: true,
+        },
+        {
+            title: "File Path",
+            dataIndex: "file_path",
+            sorter: true,
+        },
+        {
+            title: "Sample Name",
+            dataIndex: "sample_name",
+            sorter: true,
+        },
+        {
+            title: "Release Flag",
+            dataIndex: "release_flag",
+            render: (release_flag, file) => {
+                const { id } = file;
+                const options = ["", "Released", "Blocked"]
+                return <>
+                    <Select defaultValue={options[release_flag]} onChange={setReleaseFlag(id)}>
+                        {options.map((value, index) => {
+                            return <Option value={index}>{value}</Option>
+                        }).slice(1)}
+                    </Select>
+                </>
+            }
+        },
+        {
+            title: "Release Time",
+            dataIndex: "release_flag_timestamp",
+            sorter: true,
+            render: (release_flag_timestamp, _) => {
+                if (release_flag_timestamp) {
+                    const date = moment(release_flag_timestamp)
+                    return date.format("YYYY-MM-DD LT")
+                } else {
+                    return <></>
+                }
+            }
+        }
+    ]
+}
+
+const mapStateToProps = state => ({
+    datasetsById: state.datasets.itemsByID,
+    filesById: state.datasetFiles.itemsByID,
+    files: state.datasetFiles.filteredItems,
+    filters: state.datasetFiles.filters,
+    isFetching: state.datasetFiles.isFetching,
+    page: state.datasetFiles.page,
+    sortBy: state.datasetFiles.sortBy,
+    totalCount: state.datasetFiles.filteredItemsCount,
+});
+const actionCreators = {get, listFilter, update};
+
+const DatasetDetailContent = ({
+    get,
+    datasetsById,
+    files,
+    filesById,
+    isFetching,
+    listFilter,
+    page,
+    totalCount,
+    update,
+}) => {
+    const history = useHistory();
+    const {id: datasetId} = useParams();
+    const dataset = datasetsById[datasetId];
+
+    const columns = getTableColumns(
+        (id) => (release_flag) => {
+            update(id, {
+                id,
+                release_flag
+            })
+        })
+    const filterKey = DATASET_FILE_FILTERS.dataset.key
+
+    useEffect(() => {
+        if (!dataset) {
+            get(datasetId)
+        }
+    }, [datasetsById])
+
+    return <>
+    <AppPageHeader
+        title={`Dataset ${dataset?.run_name}`}
+        onBack={() => history.push("/datasets/list")}
+    />
+
+    <PageContent>
+        <Descriptions bordered={true} size="small">
```

Comment:
This is not pretty. I would try to fit all 4 items in a single row with the order of information being ID, Project, Run, Lane. If you can't fit all of them in a single row. Let the ID alone on it's row and put the other 3 on the second row.